hrms/hr/doctype/leave_application/leave_application.py

```diff
@@ -588,7 +588,7 @@ def notify_employee(self):
 			frappe.msgprint(_("Please set default template for Leave Status Notification in HR Settings."))
 			return
 		email_template = frappe.get_doc("Email Template", template)
-		message = frappe.render_template(email_template.response, args)
+		message = frappe.render_template(email_template.response_, args)
```

Comment:
The email template field is `response` and not `response_` so the current code is correct (?)